Makefile

```diff
@@ -73,6 +74,7 @@ integration-test: ## Run integration tests
 .PHONY:clean
 clean: ## Clean all artifacts from the build process
 	$(RUST_COMMAND) "--env RUST_BACKTRACE=full" "cargo clean"
+	rm -fr $(CARGO_CACHE)/*
```

Comment:
the `clean-all` command doesn't benefit from this change, but it should. The easiest thing to do is to make `clean` a dependency of `clean-all` and remove the target specific logic